packages/escalade/emulators/dosbox-sdl2/scripts/dosbox.sh

```diff
@@ -4,6 +4,6 @@
 
 emu_start
 
-dosbox "$@" > /var/log/dosbox.log 2>&1
+LANG=en_US.utf8 dosbox "$@" > /var/log/dosbox.log 2>&1
```

Comment:
This shouldn't be necessary, locale is defined in /etc/profile.d/01-locale.conf:

https://github.com/escalade/RetroELEC/blob/le9/packages/oem/filesystem/etc/profile.d/01-locale.conf

Correct spelling is en_US.UTF-8.

Reply:
LANG=en_us.uft8 was changed to LANG=C
The reason I had en_US.utf8 instead of en_US.UTF-8 is because it matched the output of `locale -a`.  However; neither LANG=en_US.UTF-8 (which is indeed already set by 01-locale.conf) nor LANG=en_US.utf8 work.